staging/src/k8s.io/api/admissionregistration/v1/types.go

```diff
@@ -563,3 +605,33 @@ type ServiceReference struct {
 	// +optional
 	Port *int32 `json:"port,omitempty" protobuf:"varint,4,opt,name=port"`
 }
+
+// MatchCondition represents a condition which must by fulfilled for a request to be sent to a webhook.
+type MatchCondition struct {
+	// Name is an identifier for this match condition, used for strategic merging of MatchConditions,
+	// as well as providing an identifier for logging purposes. A good name should be descriptive of
+	// the associated expression.
+	// Name must be a qualified name consisting of alphanumeric characters, '-', '_' or '.', and
+	// must start and end with an alphanumeric character (e.g. 'MyName',  or 'my.name',  or
+	// '123-abc', regex used for validation is '([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9]') with an
+	// optional DNS subdomain prefix and '/' (e.g. 'example.com/MyName')
+	//
+	// Required.
+	Name string `json:"name" protobuf:"bytes,1,opt,name=name"`
+
+	// Expression represents the expression which will be evaluated by CEL. Must evaluate to bool.
+	// ref: https://github.com/google/cel-spec
```

Comment:
Nit: delete this reference. It's also linked from the Kubernetes-specific documentation, and I'd prefer to just have a single reference link for this section.

Reply:
done